src/reset-local-database.sh

```diff
@@ -0,0 +1,162 @@
+      shift
+      if [[ $1 != -* ]] && [[ -n $1 ]] && test $# -gt 0; then
+        printf 'Found username argument.\n'
+        export DJANGO_SUPERUSER_USERNAME=$1
+        shift
+      else
+        printf 'No username specified after [-u|--username] flag.\n'
+        exit 1
+      fi
+      ;;
+    -p|--password)
+      shift
+      if [[ $1 != -* ]] && [[ -n $1 ]] && test $# -gt 0; then
+        printf 'Found password argument.\n'
+        export DJANGO_SUPERUSER_PASSWORD=$1
+        shift
+      else
+        printf 'No password specified after [-p|--password] flag.\n'
+        exit 1
+      fi
+      ;;
+    -e|--email)
+      shift
+      if [[ $1 != -* ]] && [[ -n $1 ]] && test $# -gt 0; then
+        printf 'Found email argument.\n'
+        export DJANGO_SUPERUSER_EMAIL=$1
+        shift
+      else
+        printf 'No email specified after [-e|--email] flag.\n'
+        exit 1
+      fi
+      ;;
+    *)
+      break
+      ;;
+  esac
+done
+
+# If the username environment variable has not been set using the arguments then check for an environment variable.
+if [[ -z "${DJANGO_SUPERUSER_USERNAME}" ]]; then
+  printf 'Please set your DJANGO_SUPERUSER_USERNAME environment variable or supply one with the [-u <username>] argument and rerun this script.\n'
+  exit 1
+fi
+
+# If the password environment variable has not been set using the arguments then check for an environment variable.
+if [[ -z "${DJANGO_SUPERUSER_PASSWORD}" ]]; then
+  printf 'Please set your DJANGO_SUPERUSER_PASSWORD environment variable or supply one with the [-p <password>] argument and rerun this script.\n'
+  exit 1
+fi
+
+# If the email environment variable has not been set using the arguments then check for an environment variable or default to a set email address.
+if [[ -z "${DJANGO_SUPERUSER_EMAIL}" ]]; then
+  printf 'No DJANGO_SUPERUSER_EMAIL environment variable or [-e <email>] argument found. Using the default "admin@example.com".\n'
+  export DJANGO_SUPERUSER_EMAIL=admin@example.com
+fi
+
+# Prompt user to confirm reset.
+read -p $'Are you sure you want to wipe your local fv_be database? [Y/N] ' yn
+case $yn in
+  [Yy]* )
+    # Drop the existing database
+    printf '\n'
+    printf 'Removing existing fv_be database.\n'
+    dropdb -f fv_be
+    retval=$?
+    if [ $retval -ne 0 ]; then
+      printf "Database removal failed: exit code $retval\n"
+      exit $retval
+    fi
+
+    # Create the empty database
+    printf '\n\n'
+    printf 'Recreating empty fv_be database.\n'
+    createdb --username=postgres fv_be
+    retval=$?
+    if [ $retval -ne 0 ]; then
+      printf "Database creation failed: exit code $retval\n"
+      exit $retval
+    fi
+    
+    # Note: this should be removed once we begin commiting migrations. See FW-4240
+    # Delete any existing backend migrations files
+    printf '\n\n'
+    printf 'Removing existing backend migrations.\n'
+    find $PWD/src/firstvoices/backend/migrations/ -not -name '__init__.py' -delete
```

Comment:
I think you should remove the src from this path. As this script is in the `src` directory, the script tries to find the migrations at ... /fv-be/src/src/firstvoices/backend/migrations/

Reply:
You should also remove the `src` from lines 108, 118, 128, 138 and 148.

Reply:
That is a good point. The intention was to run the script from the root project directory. 

I have updated the script in https://github.com/First-Peoples-Cultural-Council/fv-be/commit/6c2ac2bba7b8a681ac090b6bfed9bdb78549b150 to get the path to the script and use that, instead of using the present working directory the script was executed from. It should now work regardless of where the user runs it from.

Thanks @bronson-c!